### DISCUSSION TITLE
Crash in ph.x

### DISCUSSION THREAD

**Discussion by TheoBardu** [phoebe-team/phoebe#211] [Q&A] [answered]
Hi! Sorry for this message. I would like to know if there is some bugs in my code or if it's general. 

After installing your patched version of QE, I tried to make some calculations as test with Silicon in the EPA framework, to calculate the electronic transport properties. 
While the scf calculation works without problems, when I try to perform the phonon calculation, the program crashed, reporting this error:
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
     task #         0
     from diropn : error #        27
     error opening ./out/_ph0/Si./Users/theo/Desktop/QE/Sim/Si/test_phoebe/outPH/Si.dvscf1
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

These are my input files:
[ph.Si_in.txt](https://github.com/mir-group/phoebe/files/13874249/ph.Si_in.txt)
[pw.Si_in.txt](https://github.com/mir-group/phoebe/files/13874250/pw.Si_in.txt)

The directory /outPH exists, and inside it, the code generates the .dyn files but not the .dvscf ones. 
Is this just my problem? 
Thank you in advance.
Regards, 
Theo

**Comment by jcoulter12** [comment, marked as answer]
Hi Theo, 

Happy to have the question, I appreciate it when people report unusual behavior. 

I haven't encountered this issue before, but one thing I would say right away is that we didn't change anything in QE with respect to the reading/writing of dvscf files, which is where this call appears to come from. 

First, I would check that "./out/_ph0/Si./Users/theo/Desktop/QE/Sim/Si/test_phoebe/outPH/" is a directory you can see by typing 
`ls ./out/_ph0/Si./Users/theo/Desktop/QE/Sim/Si/test_phoebe/outPH/`

I think that QE is designed to write the dyn and dvscf files into the provided outdir. Here, you can see that the `./out/_ph0/` path is appended to your chosen prefix, `Si`, and then appended to the path you provided in fildvscf, which produces a path that doesn't exist. This makes me think the way you should set up fildyn and fildvscf as in the tutorial, by writing `fildyn = Si.dyn` and `fildvscf = Si.dvscf` and this should solve your problem. 

Let me know if this doesn't work, 
Jenny

**Reply by TheoBardu** [reply]
Thank you very much Jenny! Yeah, if I don't use any path in the definition of fildvscf it work, but it sounds strange to me, because if I define a specific path in the fildyn no errors occur, so why leave the possibility to the user to set it?
Anyway, for me this is not a problem. 
Thank you again!
Best regards,
Theo

**Reply by jcoulter12** [reply]
All good, glad it's working now!
Definitely one it's some kind of quirk in QE, which one occasionally encounters. :)
